mobile/src/main/java/rs/readahead/washington/mobile/views/fragment/vault/attachements/AttachmentsFragment.kt

```diff
@@ -826,7 +829,7 @@ class AttachmentsFragment : BaseFragment(), View.OnClickListener,
         currentMove = null
         selectMode = SelectMode.SELECT_ALL
         handleSelectMode()
-        activity.showToast(resources.getQuantityString(R.plurals.File_Successfully_Moved,filesSize))
+        showTooltip(vaultTooltip,resources.getQuantityString(R.plurals.File_Successfully_Moved,filesSize),Gravity.TOP)
```

Comment:
We should ask for clarification as to how the message should be presented by design, I don't think it's a tooltip.